diff --git a/dist/pinia.mjs b/dist/pinia.mjs
index 1087460c3e77fedefb76aaf10f81e51f96363b3a..7f91ab3989107f2e098060e88e220bf6687c9338 100644
--- a/dist/pinia.mjs
+++ b/dist/pinia.mjs
@@ -970,7 +970,7 @@ function devtoolsPlugin({ app, store, options }) {
             patchActionForGrouping(store, Object.keys(newStore._hmrPayload.actions), !!store._isOptionsAPI);
         };
     }
-    addStoreToDevtools(app, 
+    addStoreToDevtools(app,
     // FIXME: is there a way to allow the assignment from Store<Id, S, G, A> to StoreGeneric?
     store);
 }
@@ -993,7 +993,7 @@ function createPinia() {
             setActivePinia(pinia);
             pinia._a = app;
             app.provide(piniaSymbol, pinia);
-            app.config.globalProperties.$pinia = pinia;
+            // app.config.globalProperties.$pinia = pinia;
             /* istanbul ignore else */
             if ((((process.env.NODE_ENV !== 'production') || (typeof __VUE_PROD_DEVTOOLS__ !== 'undefined' && __VUE_PROD_DEVTOOLS__)) && !(process.env.NODE_ENV === 'test')) && IS_CLIENT) {
                 registerPiniaDevtools(app, pinia);
